Sources/PrimerSDK/Classes/Data Models/PrimerSessionFlow.swift

```diff
@@ -36,58 +36,59 @@
 
 public enum PrimerSessionFlow {
 
-    case `default`
-    case defaultWithVault
-    case completeDirectCheckout
-    case addPayPalToVault
-    case addCardToVault
-    case addDirectDebitToVault
-    case addKlarnaToVault
-    case addDirectDebit
+    case primerWithVault
```

Comment:
[Feedback]
We can simplify the naming of `primerWithVault` and `primerCheckout` - having `primer` as part of the name doesn't give that much information about what this does.
I think we can just call `vault` and `checkout`.

Reply:
Done

Reply:
@xevious78 I agree on what you say above. I believe the above screenshot is an outcome of not checking the flow and just adding the amount if the developer has provided one. We have to change this. I'll create a ticket for this one.

Reply:
@EvansPie I can pick this up and make sure that the flows are properly represented. Let's have a quick sync on this :)